src/evaluators/ForInStatement.js

```diff
@@ -133,6 +133,7 @@ function emitResidualLoopIfSafe(
     let absStr = realm.createAbstract(
       new TypesDomain(StringValue),
       ValuesDomain.topVal,
+      0,
```

Comment:
Some abstract values never get serialized. For these there is no need for an actual hash value.